OpenCL/m21500-pure.cl

```diff
@@ -0,0 +1,260 @@
+/**
+ * Author......: See docs/credits.txt
+ * License.....: MIT
+ */
+
+#define NEW_SIMD_CODE
+
+#ifdef KERNEL_STATIC
+#include "inc_vendor.h"
+#include "inc_types.h"
+#include "inc_platform.cl"
+#include "inc_common.cl"
+#include "inc_simd.cl"
+#include "inc_hash_sha1.cl"
+#include "inc_hash_sha512.cl"
+#endif
+
+#define COMPARE_S "inc_comp_single.cl"
+#define COMPARE_M "inc_comp_multi.cl"
+
+typedef struct pbkdf2_sha1_tmp
+{
+  u32  ipad[5];
+  u32  opad[5];
+
+  u32  dgst[260];
```

Comment:
Any reason why not to stick to 256?

Reply:
yes, without the digest buffer is not complete

Reply:
Sorry, what do you mean with "without digest buffer is not complete"?

Reply:
I mean than if you set 256 instead of 260 it don't work